ansible/config-raspberry-pi.yml

```diff
@@ -5,3 +5,4 @@
   - rad-shell
   - docker
   - pi-config
+  - boom-waffle-config
```

Comment:
This role is for boom-waffle-specific changes, not so much "general setup" kinda stuff.